uniprotkb-common/src/test/java/org/uniprot/api/uniprotkb/common/service/uniprotkb/UniProtEntryServiceTest.java

```diff
@@ -139,11 +136,10 @@ void addIsoFormFalseFilterOnlyIfQueryHasNoAccessionValue() {
                 entryService.createSearchSolrRequest(verifyFailingRequest);
         assertNotNull(verifyFailingSolrRequestCase);
         assertTrue(!verifyFailingSolrRequestCase.getFilterQueries().isEmpty());
-    }
+    }*/
 
```

Comment:
Do we not need this test which is commented?

Reply:
These tests are covered as unit test in the new RequestConverterTest and  RequestConverterUtilsTest classes